### ISSUE TITLE
Launcher should warn user before trying to run 32-bits image on a 64-bits system

### ISSUE THREAD

**Issue by demarey** [pharo-project/pharo-launcher#112]
PharoLauncher can launch any image by downloading the appropriate VM.
Nevertheless, when you use a 64-bit Pharo Launcher and try to run a 32-bit image, you will probably get an issue because Pharo needs additional setup to be able to run a 32-bit VM on a 64-bit system as listed at https://pharo.org/gnu-linux-installation : 
```bash
# Prepare 64-bit-Linux for 32bit Pharo
# The installation in Debian 8 Jessie 64bits and Ubuntu 14.04 and Later require some operations.
 sudo dpkg --add-architecture i386 
 sudo apt-get update 
 sudo apt-get install libx11-6:i386 
 sudo apt-get install libgl1-mesa-glx:i386 
 sudo apt-get install libfontconfig1:i386 
 sudo apt-get install libssl1.0.0:i386 
```

If not done, PharoLauncher will pop-up a dialog saying it cannot determine image version (uses a 32-bits VM for this purpose). This run of the VM leads to the message:
```
Error. Could not determine platform's libc path for VM. 
Try forcing $PLATFORMLIBDIR in /home/francisco/Pharo/vms/private/6521/pharo,
based on LIBC_SO.
Please report what works to pharo [vm-dev] mail list.
 LIBC_SO=
 DISTRIB_ID=Ubuntu
 DISTRIB_RELEASE=18.04
 DISTRIB_CODENAME=bionic
 DISTRIB_DESCRIPTION="Ubuntu 18.04 LTS"
 NAME="Ubuntu"
 VERSION="18.04 LTS (Bionic Beaver)"
 ID=ubuntu
 ID_LIKE=debian
 PRETTY_NAME="Ubuntu 18.04 LTS"
 VERSION_ID="18.04"
 VERSION_CODENAME=bionic
 UBUNTU_CODENAME=bionic
 UNAME=Linux Dynabook 4.15.0-22-generic #24-Ubuntu SMP Wed May 16 12:15:17
UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
 System seems to be 64 bit. You may need to (re)install the 32-bit
libraries.

Do I really need to install 32-bit libraries ? 
```

**Comment by franciscoLizarralde** [comment]
Hi demarey,
I installed the libraries (32 bits) as you suggested on my Ubuntu 18.04 (64 bits) machine and then decompressed PharoLauncher-linux-1.1-x64.zip and create a new 6.1 image.
When I tried to launch it, the PharoLauncher get closed without warning. 
I tried this twice with the same result.
I am very confused because I do not understand why if  have a 64 bit OS and use a PharoLauncher version for 64 bit and create a new image I need to install 32 bits libraries.

Please, I really need your help !!!
Thanks in advance,
Regards

**Comment by franciscoLizarralde** [comment]
Hi demarey,
I restarted the machine and PharoLauncher runs like a charm. SOLVED !!!

But could you tell me why if have a 64 bit OS and use a PharoLauncher version for 64 bit and create a new image I need to install 32 bits libraries. This drive me very confused !!!

Thanks in advance,
Regards

**Comment by estebanlm** [comment]
you are using a 32bit image as template :)

**Comment by franciscoLizarralde** [comment]
Hi Esteban,
I supposed that the image downloaded from a 64 bit PharoLauncher in a 64 bit installation would be a 64 bit image.
Thank you.

**Comment by demarey** [comment]
Yes, Pharo Launcher has to provide architecture information to avoid confusion. Indeed, if you are on a 64-bit system, it is better to use 64-bit images. Just notice that Pharo 64-bit images are quite new and not yet officially released as stable (they will be with Pharo 7.0).
Pharo Launcher let's you work with whatever image you want (32 or 64 bit)

**Comment by bencoman** [comment]
@franciscoLizarralde, 
In PharoLauncher you have two choices of Images
for Pharo 6...
a. 60540
b. 60540-64
and for Pharo 7...
c. Pharo-7.0.0-alpha.build.994.sha.6b52ae6.arch.32bit.zip
d. Pharo-7.0.0-alpha.build.994.sha.6b52ae6.arch.64bit.zip

On 64-bit Linux, extra 32-bit libs are required for (a.) & (c.).
(b.) and (d.) will work without extra libs.


@demarey,
Perhaps there can be an information pane at the bottom of PharoLauncher that displays the identified VM and extra info about libs.  The auto-VM-selection is really *fantastic*, but its a bit opaque and this could help people feel some mastery over operation of PharoLauncher.

**Comment by franciscoLizarralde** [comment]
@bencoman ,
More info about VM and extra libs would be really helpful !!!

@demarey 
Just to mention. When I run Pharo Launcher within a terminal and I try to run an Image it close without warning. This does not happen if I run Pharo Launcher without open a terminal.

**Comment by demarey** [comment]
@bencoman good idea. It would be kind of status bar. Could you open an issue for this feature?

@franciscoLizarralde And is the image being launched or not?

**Comment by bencoman** [comment]
add information pane/status bar #113

**Comment by demarey** [comment]
I will close this issue as the warning is implemented since v1.2.
Also, there is another issue to add information on the VM / launch status #113
